app/javascript/application.js

```diff
@@ -1,3 +1,2 @@
-// Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
```

Comment:
I'd lean towards keeping this comment in the source to help us find the docs in the future.